otx/mpa/seg/incremental/stage.py

```diff
@@ -0,0 +1,43 @@
+# Copyright (C) 2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+import numpy as np
+from mmcv import ConfigDict
+
+from otx.mpa.seg.stage import SegStage
+from otx.mpa.utils.config_utils import update_or_add_custom_hook
+from otx.mpa.utils.logger import get_logger
+
+logger = get_logger()
+
+
+class IncrSegStage(SegStage):
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
+
+    def configure_task(self, cfg, training, **kwargs):
+        """Adjust settings for task adaptation"""
+        super().configure_task(cfg, training, **kwargs)
+
+        self.configure_incremental(cfg, self.org_model_classes, self.model_classes)
+
+    def configure_incremental(self, cfg, org_model_classes, model_classes):
```

Comment:
I'd like to leave this method name as configure_incremental. Because it is just used when incremental learning is enabled like special sampler for incremental learning. I think configure_classes is more close to add_new_labels. I will revise it.